src/kryptos/keyContainer.js

```diff
@@ -56,8 +56,14 @@ function newKeyContainer(wrappedKey, iv, keyType) {
 }
 
 function wrapKeyContainerKey(keyToEncrypt, iv, intermediateKey) {
-  if (keyToEncrypt.privateKey) {
-    return wrapPrivateKey(keyToEncrypt.privateKey, iv, intermediateKey)
+  if (keyToEncrypt.privateKey || keyToEncrypt.extractable) {
```

Comment:
I think this part should be avoided - it's extremely hard to figure out what is happening when there are 2 options, probably exactly what you were experiencing earlier.. calling function should use privateKey or extractable...

Reply:
Agree, it now supports keyContainer key wrapping of both CryptoKey symmetric and asymmetric private key material (and custom object key material)
I left comment in code for now